realsense2_camera/launch/rs_launch.py

```diff
@@ -49,6 +49,7 @@
                            {'name': 'depth_module.infra1_format',   'default': 'Y8', 'description': 'infra1 stream format'},
                            {'name': 'depth_module.infra2_format',   'default': 'Y8', 'description': 'infra2 stream format'},
                            {'name': 'depth_module.color_profile',   'default': '0,0,0', 'description': 'Depth module color stream profile'},
```

Comment:
can you mention d405 for the depth_module.color_profile too, please?

Reply:
Done